extensions/github1s/src/providers/fileSearchProvider.ts

```diff
@@ -75,7 +75,7 @@ export class GitHub1sFileSearchProvider
 					((treeData.tree || []) as GitHubRESTEntry[]).filter(
 						(item) => item.type === 'blob'
 					),
-					{ keys: ['path'] }
+					{ keys: ['path'], threshold: 1 }
```

Comment:
Is this too loose? 

I used an alternative library https://github.com/kentcdodds/match-sorter in other projects because I like this:
`You want simple, expected, and deterministic sorting of the items (no fancy math algorithm that fancily changes the sorting as they type)`

Because I spent a ton of time tuning the Fuse.js options and couldn't find a good enough combination that is easy for users to understand and doesn't have other overhead like over matching.

Reply:
Sorry, I misunderstood the threshold option, I have tested it and it doesn't work as I expected.

I didn't know the library `match-sorted` before, it seems more appropriate in this case, I'll try this~